doc/decisions/STEPS.md

```diff
@@ -0,0 +1,144 @@
+
+The label `merge` on the edges means that a decision PR must be merged before it can target the next step.
+
+> The first PR for a decision usually creates the decision in the "Drafts" state.
+> If during the reviews of this PR it becomes clear that the decision is further along, can be moved to "Problem Clear", "Solutions Clear" or even "In Review" before the merge.
+
+Short summary:
+
+1. The first PR for a decision is created as "Drafts" (recommended!), "Problem Clear", "Solutions Clear" or "Decided".
+2. A decision PR that wants to merge a decision as "Decided" must:
+   1. Only contain changes to one decision.
+   2. Already start with the decision as "Decided".
+   3. Already have a clearly stated problem and all the solutions fully explored.
+   4. Only contain discussions/reviews about deciding between one of the solutions or details thereof.
+   5. If there is any change in the direction of the decision then the PR cannot be merged as "Decided". The decision must at least be moved back to "In Review".
+3. After a decision is merged as "Decided", it can be moved to "Partially Implemented" or "Implemented" by any PR.
+
+The rest of the document describes the steps in full details.
+
+## Drafts
+
+> This step is highly recommended and it is even required if the problem is not yet clear to all the core developers.
+
+The first step is to create a PR with:
+
+- **one** decision, where at least the "Problem" is filled out and "Decision", "Rationale" and "Implications" are **not** yet filled out.
+- optional backlinks from related decisions.
+
+This step is brainstorming for completely new ideas.
+
+> No agreement about anything is needed to merge decisions in this stage.
+
+## Problem Clear
+
+This step is very important:
+
+- it clarifies the importance of the problem, answering:
+  Why should we put our time and energy in this problem and not in another problem.
+- it clarifies the scope of the decision.
+- it clarifies relation to other problems.
+
+Decisions will have much smoother further steps if this step is done carefully without prejudice.
+It is especially important that one shouldn't have a fixed mind-set about a preferred solution from the beginning.
+
+> Everyone must agree that the problem exists and is worth solving so that a decision PR in "Problem Clear" step can be merged.
+> A problem is clear if everyone would be able to describe an experiment or test case that shows if a solution fixes the problem.
+
+## Solutions Clear
+
+> This step is recommended if it is not yet clear to the core developers which solution is the best.
+
+Here you must ensure:
+
+- problem, constraint and assumptions are well-explained and sound
+- links from/to related decisions are created
+- there are several solutions described, each with rationale and implication
+- "Decision", "Rationale" and "Implications" are **not** yet filled out if there are people arguing for different options
```

Comment:
```suggestion
- "Decision", "Rationale" and "Implications" are **not** yet filled out
```

If people don't argue for different options, this would be completely the same as "In Review". So better just say "not filled out at all" here.